### PR TITLE
Add condensed tutorial & team bio

### PR DESCRIPTION
**Pull request by adelavega**
* Added Team bio and pictures for all
* Added a condensed version of the OHBM tutorial for "Manual Meta-Analysis". The idea is to explain the process from end to end in as short of a format as possible. I link to the original tutorial for those with more time.

### CODE REVIEW

**Comment by adelavega** [comment]
@jdkent @nicoalee please review!

**Comment by nicoalee** [comment]
some of the tutorial steps are a bit out of date as we've changed some features (like the banner that reads CLONE THIS STUDY) for example, but overall it looks great.

**Review by nicoalee — APPROVED** [review]

File: static/tutorial/annotate_and_extract_study.png
Comment (nicoalee) [inline static/tutorial/annotate_and_extract_study.png:1]: this is now out of date

File: static/tutorial/create_project.png
Comment (nicoalee) [inline static/tutorial/create_project.png:1]: we now auto create the project without having the user add a name or description. They can add it later if they want

File: static/tutorial/display_imported_stubs_1.png
Comment (nicoalee) [inline static/tutorial/display_imported_stubs_1.png:1]: this will change based on feedback

**Comment by adelavega** [comment]
Oh-- some of the screenshots I took straight from Jame's, so I can definitely update those

**Pull request merged**